WordPress/src/main/res/layout/login_epilogue_header.xml

```diff
@@ -45,13 +45,13 @@
             android:paddingStart="@dimen/margin_extra_large"
             android:paddingEnd="@dimen/margin_extra_large">
 
-            <org.wordpress.android.widgets.WPNetworkImageView
+            <ImageView
                 android:id="@+id/avatar"
                 android:layout_width="@dimen/avatar_sz_inner_circle"
                 android:layout_height="@dimen/avatar_sz_inner_circle"
                 android:layout_marginTop="@dimen/margin_extra_medium_large"
                 android:layout_marginBottom="@dimen/margin_medium"
-                app:wpDefaultImageDrawable="@drawable/ic_gridicons_user_circle_100dp"/>
```

Comment:
Is it ok that we're removing the default image here?

Reply:
All the placeholder are encapsulated in `ImagePlaceholderManager`, which is used in `ImageManager`.